src/OrchardCore/OrchardCore.ContentManagement/DefaultContentManager.cs

```diff
@@ -365,7 +365,7 @@ protected async Task<ContentItem> BuildNewVersionAsync(ContentItem existingConte
             return context.BuildingContentItem;
         }
 
-        public async Task CreateAsync(ContentItem contentItem, VersionOptions options)
+        public async Task CreateAsync(ContentItem contentItem, VersionOptions options, bool invokeUpdateCallbacks = false)
```

Comment:
I am the API police ;) Why not call `UpdateAsync` after `CreateAsync`, I think that's why we created `UpdateAsync`.

Reply:
We just save a double call to session.save.

Reply:
The only difference is that one will trigger the events from UpdateAsync and the other not.

Reply:
That's all? Then the second call is no-op in YesSql, so it's not really saving anything

Reply:
Save is just 'marking' an entity in yessql, it doesn't do anything when it's called.

Reply:
@MichaelPetrinolis please give me a Worflow example so that I can test it to make this faster. I can create a script task with a createContentItem function.
I don't think it would/should be any different than when calling a createContentItem task.

https://docs.orchardcore.net/en/dev/docs/reference/modules/Scripting/#content-orchardcorecontents
vs
https://docs.orchardcore.net/en/dev/docs/reference/modules/Workflows/#activities-out-of-the-box